### PR TITLE
Running local test creates excessive image rendering.

### PR DESCRIPTION
**Pull request by techcobweb**
Signed-off-by: Mike Cobbett <77053+techcobweb@users.noreply.github.com>

- bootstrap.properties comments has a small bug inside. Fixed.
- build-locally.sh script runs 2 simple tests to up the unit test for extra coverage, instead of 1.
- found lots of code where err was being checked but we already know it's nill or non-nil.
- found some code which would trap. eg: err.Error() when we already know err is nil.
- found some unused parameters. 2 I commented, as i'm unsure why the code is like it is... think there is something wrong in those two areas ?
- image rendering only happens once after a local run, against the entire home folder.
- messages suppressed to make logs more readable.
- removed unused parameters in some places.
- Local runs : images are rendered only on the runs folders which were created by the runs.
- Download --force allows images to be overwritten.

### CODE REVIEW

File: pkg/runs/submitter.go [hunk 1]
```diff
@@ -282,7 +282,7 @@ func (submitter *Submitter) submitRun(
 	readyRuns []TestRun,
 	submittedRuns map[string]*TestRun,
 	lostRuns map[string]*TestRun,
-	runOverrides *map[string]string,
+	runOverrides *map[string]string, // This doesn't appear to be used. Why not ?
```
Comment (KirbyKatcher) [inline pkg/runs/submitter.go:285]: I think they are used by the jvmlauncher, if they are not then we have a pretty big bug

File: pkg/runs/testSelection.go [hunk 2]
```diff
@@ -304,7 +302,8 @@ func selectTestsByTest(testCatalog launcher.TestCatalog, testSelection *TestSele
 	return err
 }
 
-func selectTestsByClass(testCatalog launcher.TestCatalog, testSelection *TestSelection, flags *utils.TestSelectionFlagValues) error {
+func selectTestsByClass(testCatalog launcher.TestCatalog, // Shouldn't this be used ?
```
Comment (KirbyKatcher) [inline pkg/runs/testSelection.go:305]: This is being used on line 188 of this file

**Review by KirbyKatcher — APPROVED** [review]

File: pkg/runs/testSelection.go (hunk 2 above)
Comment (techcobweb) [reply pkg/runs/testSelection.go:305]: I was talking about the parameter, not the method.

File: pkg/runs/submitter.go (hunk 1 above)
Comment (techcobweb) [reply pkg/runs/submitter.go:285]: No. It's not used. Does that mean overrides are just ignored ?

**Comment by galasa-team** [comment]
Build successful

File: pkg/runs/submitter.go (hunk 1 above)
Comment (techcobweb) [reply pkg/runs/submitter.go:310]: Raised a separate issue to track this.

**Pull request merged**